internal/services/recoveryservices/site_recovery_replication_policy_hyperv_resource.go

```diff
@@ -0,0 +1,244 @@
+package recoveryservices
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/recoveryservicessiterecovery/2022-10-01/replicationpolicies"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/recoveryservices/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type ReplicationPolicyHyperVModel struct {
+	Name                                          string `tfschema:"name"`
+	ResourceGroupName                             string `tfschema:"resource_group_name"`
+	RecoveryVaultName                             string `tfschema:"recovery_vault_name"`
```

Comment:
We can replace these with one property: 
```suggestion
	RecoveryVaultId                             string `tfschema:"recovery_vault_id"`
```